extmod/btstack/btstack_config_common.h

```diff
@@ -35,7 +35,7 @@
 #define MAX_NR_LE_DEVICE_DB_ENTRIES 4
 
 // Link Key DB and LE Device DB using TLV on top of Flash Sector interface
-// #define NVM_NUM_DEVICE_DB_ENTRIES 16
+#define NVM_NUM_DEVICE_DB_ENTRIES 16
```

Comment:
With this here, the event `_IRQ_GET_SECRET` is called 16 times, with the string `<n>DBT` for n in 0..15 inclusive.  I'm not sure exactly what this is needed for, do you know?

Reply:
This is the le_device_db implementation of btstack: https://github.com/bluekitchen/btstack/blob/2b49e57bd1fae85ac32ac1f41cdb7c794de335f6/src/ble/le_device_db_tlv.c#L208. Basically it implements the device db as a (fixed) list of device, and this code scans through all slots to find the least recently used empty slot.